src/styles/SearchForm.css

```
@@ -0,0 +1,5 @@
+
+button{
```

```suggestion
button {
```

There are multiple instances of using the opening bracket straight after the class name throughout, there should be a space included